docs/cloud-api/user-guide.md

```diff
@@ -0,0 +1,71 @@
+# User guide
+
+This guide will illustrate the steps required for configuring an application that uses the new Cloud Connection APIs to publish messages to the Kapua platform.
+
+The involved steps are the following
+
+1. [Instantiation and configuration of the Cloud Connection](#creating-a-new-cloud-connection).
+2. [Instantiation and configuration of a Publisher](#creating-and-configuring-a-new-publisher).
+3. [Binding an application to the Publisher](#binding-an-application-to-a-publisher).
+
+## Creating a new Cloud Connection
+
+1. Open the Cloud Connections section of the Web UI:
```

Comment:
Need to indent the image to have the enumeration working for the list. Please apply to the rest of the file